nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -152,7 +152,8 @@ language governing permissions and limitations under the License. -->
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-client</artifactId>
+            <artifactId>activemq-all</artifactId>
+            <scope>test</scope>
```

Comment:
Yeah, activemq-all will bring the client back in. The real issue is that when using embedded broker like I am doing in test you also need other JARs from active mq and they all have to be in sink. So activemq-all will ensure that. Also, we don't need it at runtime, only at testCompile